packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -776,8 +776,8 @@ class RenderFlex extends RenderBox with ContainerRenderObjectMixin<RenderBox, Fl
   Rect describeApproximatePaintClip(RenderObject child) => _overflow > 0.0 ? Offset.zero & size : null;
 
   @override
-  String toString() {
-    String header = super.toString();
+  String toShortDescription() {
```

Comment:
I think we're probably better off having the toString return the short description of a RenderObject, because RenderObjects' string descriptions don't generally fit on one line but the render object toString is frequently used in the middle of paragraphs and such.